README.md

```diff
@@ -1,2 +1,37 @@
-# template-public
-general template for public repositories
+# sync-s3-to-b2
+Service to synchronize an AWS S3 bucket path to a Backblaze B2 bucket path
+
+This process uses `rclone sync` to copy changes from a specified directory in an AWS S3 bucket to a specified directory in a Backblaze B2 bucket. After configuring `rclone`, the script executes `rclone sync --checksum --create-empty-src-dirs --metadata --transfers 32 ${RCLONE_ARGUMENTS} s3bucket:${S3_BUCKET}/${S3_PATH} b2bucket:${B2_BUCKET}/${B2_PATH}`.
```

Comment:
Is this level of detail helpful enough to risk becoming out of sync with the code?   Maybe it's enough to say something like "After configuring rclone, the script executes rclone to synchronize the configured buckets."

Reply:
This ties into my validation of environment variables comment indicating a way to improve the script.

Reply:
I'd rather read two English sentences that summarize what the code does, even if the included command is out of date, than comprehend 74 lines of shell script only to discover that the `rclone sync` command won't work for me.

Reply:
I think we're in agreement, then. Keep the English, toss the script command. Right?

Reply:
I don't think we are.  If the `rclone sync` command isn't in the README, then the user is forced to read 74 lines of shell script before he finds out how the synchronization is accomplished.